setup.py

```diff
@@ -186,6 +186,8 @@ def run(self):
   libs.extend(['onnxruntime_providers_tensorrt.dll'])
   # DirectML Libs
   libs.extend(['directml.dll'])
+  # OpenVINO Libs
+  libs.extend(['onnxruntime_providers_openvino.dll'])
```

Comment:
same comment as above. and should we add for Mac OSX?

Reply:
We don't support Mac OSX yet. So not adding one for now.

Reply:
ok. but openvino does support mac osx. we can add an item to backlog.

Reply:
yes, we will.